src/Raven.Client/Documents/Linq/RavenQueryProvider.cs

```diff
@@ -31,18 +32,22 @@ internal class RavenQueryProvider<T> : IRavenQueryProvider
         private readonly QueryStatistics _queryStatistics;
         private readonly LinqQueryHighlightings _highlightings;
         private readonly bool _isMapReduce;
+        private DocumentConventions _conventions;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="RavenQueryProvider{T}"/> class.
         /// </summary>
         public RavenQueryProvider(
+#if FEATURE_HIGHLIGHTING
```

Comment:
Why is this under #if def?

Reply:
It needs to be removed entirely. Probably git cherry-pick issue.